Source/FakeItEasy.Specs/UnconfiguredFakeSpecs.cs

```diff
@@ -6,23 +6,28 @@
 
     public static class UnconfiguredFakeSpecs
     {
+        public interface IFoo
+        {
+            bool IsADummy { get; }
+        }
+
         [Scenario]
         public static void VirtualMethod(
             MakesVirtualCallInConstructor fake)
         {
-            "when faking a class with a virtual method"
+            "When faking a class with a virtual method"
                 .x(() => fake = A.Fake<MakesVirtualCallInConstructor>());
 
-            "it should return a default value when the method is called during the constructor"
+            "It should return a default value when the method is called during the constructor"

```

Comment:
> on the other hand a When which so long that it scrolls off the side of the GitHub UI isn't very desirable

Oh. Is that your concern?
"Given an unconfigured fake that assigns values to a virtual reference type property during construction"